Makefile

```diff
@@ -74,6 +74,7 @@ DOCBOOK2PDF ?= docbook2pdf
 
 LCOV ?= lcov
 GENHTML ?= genhtml
+SED ?= sed
```

Comment:
OK, looks like this was a bad idea. I'll reenable my `sed -> gsed` alias and maybe configure a build that runs on macOS to ensure that the portability fixes you did in the test suite won't regress.